specs/daito/OpenAPI_Design.md

```diff
@@ -0,0 +1,164 @@
+# VMware Ecosystem OpenAPI
+
+**Author(s)**: [zengyingzhe](https://github.com/zengyingzhe)
+
+## Summary
+
+This documentation serves as the design spec for VMware OpenAPI.
```

Comment:
Can we call it "OpenSDS Storage Provider for VMware " . Because in VMware terms A software component developed by a third party through vSphere APIs for Storage Awareness (VASA) is called storage Provider.

Reply:
Thanks for the comment.
"Provider" sounds more like a server, such as the VASA provider.
However, openapi is just a library here, providing normal interfaces to VMware plugins to call.